core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -139,6 +198,144 @@ var StoryEditorPage = function() {
     closeSaveModalButton.click();
     waitFor.pageToFullyLoad();
   };
+
+  this.expectSaveStoryDisabled = function() {
+    return expect(
+      saveStoryButton.getAttribute('disabled')).toEqual('true');
+  };
+
+  this.expectDisplayUnreachableChapter = function() {
```

Comment:
Could be confused with expecting that an unreachable chapter is displayed. Maybe `expectDisplayUnreachableChapterWarning`?

Reply:
Updated